src/presentation/wallet/send/lightning-enter-invoice.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router';
+import ShellPopUp from '../../components/shell-popup';
+import cx from 'classnames';
+import { NetworkString } from 'ldk';
+import Button from '../../components/button';
+import { fromSatoshi } from '../../utils/format';
+import { selectMainAccount } from '../../../application/redux/selectors/wallet.selector';
+import { useDispatch, useSelector } from 'react-redux';
+import { ProxyStoreDispatch } from '../../../application/redux/proxyStore';
+import {
+  incrementAddressIndex,
+  incrementChangeAddressIndex,
+} from '../../../application/redux/actions/wallet';
+import { isSet } from '../../../application/utils/common';
+import { DEFAULT_LIGHTNING_LIMITS, getInvoiceValue, isValidSubmarineSwap } from '../../utils/boltz';
+import Boltz, { SubmarineSwapResponse } from '../../../application/utils/boltz';
+import { createAddress } from '../../../domain/address';
+import {
+  setAddressesAndAmount,
+  setPendingTxStep,
+} from '../../../application/redux/actions/transaction';
+import { SEND_CHOOSE_FEE_ROUTE } from '../../routes/constants';
+import { BalancesByAsset } from '../../../application/redux/selectors/balance.selector';
+import { lbtcAssetByNetwork } from '../../../application/utils/network';
+
+export interface LightningInvoiceProps {
+  balances: BalancesByAsset;
+  network: NetworkString;
+}
+
+const LightningInvoiceView: React.FC<LightningInvoiceProps> = ({ balances, network }) => {
+  const dispatch = useDispatch<ProxyStoreDispatch>();
+
+  const history = useHistory();
+  const account = useSelector(selectMainAccount);
+
+  const [error, setError] = useState('');
+  const [invoice, setInvoice] = useState('');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [limits, setLimits] = useState(DEFAULT_LIGHTNING_LIMITS);
+  const [touched, setTouched] = useState(false);
+  const [value, setValue] = useState(0);
+
+  const lbtcAssetHash = lbtcAssetByNetwork(network);
+  const lbtcBalance = balances[lbtcAssetHash];
```

Comment:
Nit: container could directly return the lbtc balance (instead of all ones)

Reply:
Fixed by 140a7e5